beater/server_test.go

```diff
@@ -74,12 +74,28 @@ func TestServerFrontendSwitch(t *testing.T) {
 	apm.Handler.ServeHTTP(rec, req)
 	assert.Equal(t, 403, rec.Code, rec.Body.String())
 
-	apm.Handler = newMuxer(Config{EnableFrontend: true}, nil)
+	apm.Handler = newMuxer(Config{EnableFrontend: true, AllowOrigins: []string{"*"}}, nil)
```

Comment:
As you want to test the `FrontendSwitchHandler` I suggest to also set `AllowOrigins` for the test that returns a 403. Otherwise it could be that it is forbidden because of the origin.